spec/support/test_setup.rb

```diff
@@ -2,118 +2,54 @@
 class TestSetup
```

Comment:
I'm thinking we should rename this to `TestDataCache`

Reply:
Yeah that seems reasonable. Or more more simply 'TestData'? What do others think?